metricbeat/module/nats/connections/_meta/fields.yml

```diff
@@ -0,0 +1,17 @@
+- name: connections
+  type: group
+  description: >
+    Contains nats connection related metrics
+  fields:
+    - name: server_id
```

Comment:
As we have this field also in the stats module, I think we should move it one level up to the module level to only 1 field. I assume it's the same?